README.md

```diff
@@ -14,6 +14,23 @@ If you use a non-supported version of TypeScript, the parser will log a warning
 
 **Please ensure that you are using a supported version before submitting any issues/bug reports.**
 
+## Known Issues
+
+The following ESLint rules will fail on acceptable code:
+ - no-undef [#77](https://github.com/eslint/typescript-eslint-parser/issues/77)
+ - no-unused-vars [#77](https://github.com/eslint/typescript-eslint-parser/issues/77)
+ - no-useless-constructor [#77](https://github.com/eslint/typescript-eslint-parser/issues/77)
+ - space-infix-ops [#224](https://github.com/eslint/typescript-eslint-parser/issues/224)
+ 
+The follow ESLint plugins have issues when used with this parser:
+ - eslint-plugin-react [#213](https://github.com/eslint/typescript-eslint-parser/issues/213)
+  
+The following TypeScript syntax will cause rules to fail or ESLint v3 to crash:
+ - Empty body functions
+   - Abstract methods
+   - Multiple function declarations
```

Comment:
https://github.com/eslint/typescript-eslint-parser/issues/253 Summarizes the current situation and the three PR's I created to fix this issue.